server/adapter/handler/websocket_handler.go

```diff
@@ -83,7 +85,19 @@ func (h *WebSocketHandler) NotifyWebSocket(c echo.Context) error {
 		return err
 	}
 
-	client := entity.NewNotifyClientEntity(conn, userId)
+	joinedChannelIDs, err := h.channelInteractor.GetJoinedChannelIDs(ctx, userId)
+	if err != nil {
+		cancel()
+		return err
+	}
+
+	// NOTE: 通知クライアントが参加しているチャンネルを取得
+	joinedChannels := make(map[uint64]*entity.NotifyClientJoinedChannel)
+	for _, channelID := range joinedChannelIDs {
+		joinedChannels[channelID] = entity.NewNotifyClientJoinedChannelEntity(true)
+	}
+
+	client := entity.NewNotifyClientEntity(conn, userId, joinedChannels)
```

Comment:
```suggestion
	client := entity.NewNotifyClientEntity(conn, userId, joinedChannelIDs)
```

にしてNotifyClientJoinedChannelはここで生成する必要がなさそう